contracts/BSCValidatorSet.sol

```diff
@@ -631,12 +568,7 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
     }
 
     function removeTmpMigratedValidator(address validator) external onlyStakeHub {
-        for (uint256 i; i < _tmpMigratedValidatorSet.length; ++i) {
-            if (_tmpMigratedValidatorSet[i].consensusAddress == validator) {
-                _tmpMigratedValidatorSet[i].jailed = true;
-                break;
-            }
-        }
+        revert("deprecated");
     }
 
     /*----------------- For Temporary Maintenance -----------------*/
```

Comment:
in the `updateParam`, some param no need to be update any longer, like `expireTimeSecondGap`, so please delete them

Reply:
removed

Reply:
removed